### PR TITLE
checks if new version of flags exists every 2 mins, then downloads an…

### PR DESCRIPTION
**Pull request by leggsimon**
…y changes

### CODE REVIEW

File: client/js/data/api.js [hunk 1]
```diff
@@ -119,6 +122,18 @@ function getAll(){
 		.catch(error);
 }
 
+function getAllInBackground() {
+	console.log("Checking if flags have been updated...");
+	return makeApiCall('GET', '/')
+		.then(function(data) {
+			if (data.version > currentDataVersion || !currentDataVersion) {
+				showOverlay('Loading flags');
+				console.log("Flags v" + currentDataVersion + " updated to v" + data.version);
```
Comment (ironsidevsquincy) [inline client/js/data/api.js]: use https://github.com/Financial-Times/next-logger for logging

File: client/js/data/api.js [hunk 2]
```diff
@@ -119,6 +122,18 @@ function getAll(){
 		.catch(error);
 }
 
+function getAllInBackground() {
+	console.log("Checking if flags have been updated...");
+	return makeApiCall('GET', '/')
+		.then(function(data) {
+			if (data.version > currentDataVersion || !currentDataVersion) {
+				showOverlay('Loading flags');
```
Comment (aintgoin2goa) [inline client/js/data/api.js]: by this point the flags have already been loaded so there's not really any benefit in showing the overlay now.

File: client/js/data/api.js (hunk 1 above)
Comment (aintgoin2goa) [inline client/js/data/api.js]: does that work client-side as well?

Comment (ironsidevsquincy) [inline client/js/data/api.js]: durrrrrr. ignore me

**Comment by aintgoin2goa** [comment]
:+1:

**Pull request merged**